MAINTAINERS.md

```diff
@@ -0,0 +1,14 @@
+At present the maintainers are (alphabetically):
+
+* David Fisher (@ddfisher)
```

Comment:
David Fisher and Matthias Kramm do not actually have access to the repo.

People not listed here who do have access include Michael Lee, Michael Sullivan, Rebecca Chen, and Andrey Vlasovskikh.

Reply:
I assume (but I am not sure) that David and Matthias have access through the Python organization, but I have no access to the member list to confirm. Andrey also should have access through that. We should probably review the member list and maintainer list at some point. But that's a separate issue.

Reply:
They don't; they are not members of the typeshed team. But agree that this can be improved separately.